api/v1/operator/openapi.yaml

```diff
@@ -0,0 +1,54 @@
+---
+swagger: '2.0'
+info:
+  title: Cilium Operator
+  description: Cilium
+  version: v1beta
+x-schemes:
+- unix
+basePath: "/v1"
+produces:
+- application/json
+consumes:
+- application/json
+paths:
+  "/metrics/":
+    get:
+      summary: Retrieve cilium operator metrics
+      tags:
+      - metrics
+      parameters: []
+      responses:
+        '200':
+          description: Success
+          schema:
+            type: array
+            items:
+              "$ref": "../openapi.yaml#/definitions/Metric"
+        '500':
+          x-go-name: Failed
+          description: Metrics cannot be retrieved
+  "/healthz":
+    get:
+      summary: Get health of Cilium operator
+      description: This path will return the status of cilium operator instance.
+      tags:
+      - operator
+      parameters: []
+      responses:
+        '200':
+          description: Cilium operator is healthy
+          schema:
+            "$ref": "#/definitions/StatusResponse"
```

Comment:
This should not have a schema.

Reply:
Ok, I did not realize that we need to preserve the previously used plain text schema for `/healthz`